tests/Tags/GlideTest.php

```diff
@@ -67,6 +67,20 @@ public function it_outputs_a_data_url()
         $this->assertStringStartsWith('data:image/jpeg;base64', (string) Parse::template($tag, ['foo' => 'bar.jpg']));
     }
 
+    #[Test]
+    #[DefineEnvironment('absoluteHttpRouteUrlWithoutCache')]
```

Comment:
Why do we need to bypass the Glide cache here? The test still seems to pass without it. 🤔